package.json

```diff
@@ -118,7 +118,9 @@
   "types": "index.d.ts",
   "typesVersions": {
     "<3.9": {
-      "*": "ts3.4/*"
+      "*": [
+        "ts3.4/*"
+      ]
```

Comment:
`typesVersions` uses the same format as `paths` with an array and silently does nothing if any other kind of value is specified 😬

Reply:
> silently does nothing

GAAAAH

Thanks for catching this.